10up/ElasticPressLabs
# Moving geolocation feature to ElasticPressLabs

### Description of the Change

This change adds a new Feature that supports geolocation.

Ref: https://github.com/10up/ElasticPress/pull/2096

### Alternate Designs

### Benefits

### Possible Drawbacks

### Verification Process

1. In the Elasticpress dashboard, enable the feature Geolocation Search; 
2. Set the "Location Me" selector field and check if you can see the button "Use My Location";
3. Create two meta fields for a post with the name `latitude` and `longitude` and set the values with your latitude and longitude.
4. Run an index;
5. On the site, click on the button "Use My Location";
6. Search for some term that appears in the post used in step 3 and the result should be only this post;
7. Click on "Stop using my location", search again using the same term and the result may include other posts; 

### Checklist:




### Applicable Issues

### Changelog Entry

Added new Feature that supports geolocation
